napari/components/experimental/_plane.py

```diff
@@ -0,0 +1,28 @@
+from typing import Tuple
```

Comment:
Maybe we put this in a `napari/layers/utils` rather than `napari/components/experimental`, unless you think it will also be used by the viewer directly sometime instead of just our layers.

Also if we expose the object at `layer.plane` then i think no need to have the object inside an `experimental` folder

Reply:
FYI, this is code form #2074, which may be worth discussing/merging separately.

Reply:
ah my bad - though did you mean napari #2952? yes maybe let's review that first, my bad for not catching that!!

Reply:
How on earth did 2952 become 2074? :rofl: Yes, I meant that!